docs/development_guide/targets.rst

```diff
@@ -3,7 +3,14 @@ Targets
 
 To facilitate encapsulation and reuse of schema parameters related to design targets, SiliconCompiler implements a :meth:`.load_target()` function which can run scripts that set up common combinations of :class:`.Flow`, :class:`.PDK`, :class:`.Library`, and :class:`.Checklist` modules.
 
-The :meth:`.load_target()` function takes in a string ``targetname``. Similar to most other SiliconCompiler modules, it will search for a ``targets.targetname`` module in the current directory and Python search path. A full list of built-in targets can be found on the :ref:`targets_directory` page.
+SiliconCompiler comes with a set of built-in targets. If you want to load a built-in target module, you can import it as a package:
+
+.. code-block:: Python
+  
+  from siliconcompiler.targets import <pre-built targetname>
```

Comment:
you still need to add the example code to pass this to `load_target` to make this a complete example